api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/controllers/api/ServicesController.java

```diff
@@ -155,56 +179,31 @@ private String getApiDoc(String serviceId) {
         @ApiResponse(responseCode = "404", description = "URI not found"),
         @ApiResponse(responseCode = "500", description = "An unexpected condition occurred")
     })
-    public ResponseEntity<APIService> getAPIServicesById(@PathVariable(value = "id") String id) throws ContainerStatusRetrievalException {
-        try {
-            var service = containerService.getService(id);
-            if (service == null) {
-                return new ResponseEntity<>(HttpStatus.NOT_FOUND);
-            }
-            log.debug("Getting service api doc by id {}", id);
-            String apiDoc = getApiDoc(id);
+    public Mono<ResponseEntity<APIService>> getAPIServicesById(@PathVariable(value = "id") String id) throws ContainerStatusRetrievalException {
```

Comment:
Exception 'org.zowe.apiml.apicatalog.exceptions.ContainerStatusRetrievalException' is never thrown in the method

Reply:
removed